docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,266 @@
+
+	source ~/.bash_profile
+
+That's it. Bash Completion is enabled.
+
+## Basics of WP-CLI
+Before we get into the helm of the things, let us learn some essential basics of how exactly WP-CLI works. This would help you feel comfortable with what is going to follow next.
+
+So far we have seen WP-CLI is accessed through the keyword `wp`. What follows that is a set of commands and their own sub commands. For example we have a command to download WordPress which goes like `wp core download`. Here wp is the main command while core and download are its sub-commands. There can be several sub-commands for each sub-command as well.
+
+WP-CLI also comes with a pretty detailed help section which will allow you to keep tab on all the commands you would need. You can access it by typing
+
+	wp help
+	
+Doing that and you will see a screen similar to
+
+	wp
+
+	DESCRIPTION
+
+	Manage WordPress through the command-line.
+
+	SYNOPSIS
+
+	wp <command>
+
+	SUBCOMMANDS
+
+	cache               Manage the object cache.
+	cap                 Manage user capabilities.
+	cli                 Get information about WP-CLI itself.
+	comment             Manage comments.
+	core                Download, install, update and otherwise manage WordPress proper.
+	cron                Manage WP-Cron events and schedules.
+	db                  Perform basic database operations.
+	eval                Execute arbitrary PHP code after loading WordPress.
+	eval-file           Load and execute a PHP file after loading WordPress.
+	:
+	
+	
+`:` is a prompt where you need to enter few commands to navigate through this help menu. Up and down arrow keys will take you through the complete help. And pressing q will exit the help menu. That's all you need to know for now. For complete detail on how to navigate through the complete help section, you can always type h at the above prompt.
+
+In our last step, we enabled Bash Completion feature for WP-CLI. To use that type `wp` and press tab twice. You would see the list of available commands. You can repeat the same by typing 'wp core' and then pressing tab twice. Now you will see a list of commands that can be used with `core`.
+
+## Installing WordPress
+
+### Seting up Database
+
+Before you proceed, you need to setup a database first. For that, login to the MySQL server first.
+
+	mysql -u user -p
+
+Next step is to create a database.
+
+	create database wordpress;
+
+Now we need to grant all the required priviges for the database to the mysql user.
+
+	grant all on wordpress.* to 'user' identified by 'password';
+
+Type `quit` to exit the MySQL commandline.
+
+First of all, move to the user directory where you would be installing the blog. Traditionally it should be in `/var/www/html` directory. Create a new user directory for your user if it doesn't exist already. Also proper group and ownership permissions need to be given to the user.
+
+	cd /var/wwww/html
+	sudo mkdir wpblog
+	sudo chown -R user:user wpblog
```

Comment:
Can you please explain your decision to give `user` permissions over the WordPress directory? 

Also, is `wpblog` a standard example name?